baselines/compute/samples/generated/v1/snippet_metadata.addresses.v1.json.baseline

```diff
@@ -0,0 +1,243 @@
+{
+  "clientLibrary": {
+    "name": "nodejs-compute",
+    "version": "0.1.0",
+    "language": "TYPESCRIPT",
+    "apis": [
+      {
+        "id": "Addresses",
```

Comment:
This should be the proto package.
I'm guessing this is really just the Compute API?, and maybe they should all go together in a single file?

The same as the file name, it should be snippet_metadata_{protopackage}.json. 

Or what am I missing?